security/zero_trust_workload_identity_manager/zero-trust-manager-oidc-federation.adoc

```diff
@@ -0,0 +1,39 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="zero-trust-manager-oidc-federation"]
+= {zero-trust-full} OIDC Federation
+include::_attributes/common-attributes.adoc[]
+:context: zero-trust-manager-oidc-federation
+
+
+toc::[]
+
+:FeatureName: Zero Trust Workload Identity Manager for Red{nbsp}Hat OpenShift
+
+include::snippets/technology-preview.adoc[]
+
+{zero-trust-full} integrates with OpenID Connect (OIDC) by allowing a SPIRE server to act as an OIDC provider. This enables workloads to request and receive verifiable JSON Web Tokens-SPIFFE Verifiable Identity Documents (JWT-SVIDS) from the local SPIRE agent. External systems, such as cloud providers, can then use the OIDC discovery endpoint exposed by the SPIRE server to retrieve public keys.
+
+The following OIDC protocols are available:
+
+* Vault
+
+* EntraID
```

Comment:
```suggestion
* Vault

* Azure Entra ID

* AWS

* GCP
```

AWS and GCP are also verified, although we don't need to document the exhaustive steps in this PR.

Reply:
Okay, let's keep it as-is for now